numpy/add_newdocs.py

```diff
@@ -3096,18 +3096,10 @@ def luf(lamdaexpr, *args, **kwargs):
 
 
 add_newdoc('numpy.core.multiarray', 'ndarray', ('__copy__',
-    """a.__copy__([order])
-
-    Return a copy of the array.
-
-    Parameters
-    ----------
-    order : {'C', 'F', 'A'}, optional
-        If order is 'C' (False) then the result is contiguous (default).
-        If order is 'Fortran' (True) then the result has fortran order.
-        If order is 'Any' (None) then the result has fortran order
-        only if the array already is in fortran order.
+    """a.__copy__()
 
+    Used if copy.copy is called on an array. Returns a copy of the array.
```

Comment:
Might be worth mentioning that the behaviour is `order='K'`

Reply:
Done!